src/header/header.scss

```diff
@@ -274,61 +227,72 @@
 
   .iui-header-button-icon {
     padding: $iui-xs;
+    box-sizing: content-box;
     @include iui-header-button-icon($iui-icons-default, $iui-border-radius);
     @include themed {
       background-color: t(iui-color-background-3);
       fill: t(iui-icons-color);
     }
   }
 
+  img.iui-header-button-icon {
+    padding: 0;
+    width: auto;
+    height: $iui-icons-default + ($iui-xs * 2);
+  }
+
   &.iui-active {
```

Comment:
Maybe we could have a single set of active rules?
![image](https://user-images.githubusercontent.com/81580355/153180059-23f1aef7-6265-4625-9219-f8b6bd35746c.png)

Reply:
Also this one looks weird..
![image](https://user-images.githubusercontent.com/81580355/153849562-c5c9af64-a09f-4d66-b35d-624dd45ad65f.png)